### PR TITLE
Support canvas tooltips in API 1.12

### PR DESCRIPTION
**Pull request by jennyfel**

### CODE REVIEW

File: templates/visuals/.api/v1.12.0/schema.capabilities.json [hunk 1]
```diff
@@ -667,6 +670,33 @@
             },
             "additionalProperties": false
         },
+        "tooltips": {
+            "type": "object",
+            "description": "Tells the host to include tooltips ability",
```
Comment (AviSander) [inline templates/visuals/.api/v1.12.0/schema.capabilities.json]: Tells -> instructs

File: templates/visuals/.api/v1.12.0/schema.capabilities.json [hunk 2]
```diff
@@ -667,6 +670,33 @@
...
+            "properties": {
+                "supportedTypes": {
+                    "type": "object",
+                    "description": "Tells the host what tooltip types to support",
+                    "properties": {
+                    "default": {
+                        "type": "boolean",
+                        "description": "Tells the host to support showing default tooltips"
+                        },
```
Comment (AsafMozes) [inline templates/visuals/.api/v1.12.0/schema.capabilities.json]: Indentation looks a bit off. Please run some re-formatter on the new section.

File: templates/visuals/.api/v1.12.0/schema.capabilities.json (hunk 1 above)
Comment (jennyfel) [reply templates/visuals/.api/v1.12.0/schema.capabilities.json]: Done

File: templates/visuals/.api/v1.12.0/schema.capabilities.json (hunk 2 above)
Comment (jennyfel) [reply templates/visuals/.api/v1.12.0/schema.capabilities.json]: Done

**Pull request merged**